### ISSUE TITLE
Guide for diagnosing excess graph breaks

### ISSUE THREAD

**Issue by ezyang** [pytorch/torchdynamo#679]
Something a user using torchdynamo might want to do is figure out why they are having lots of graph breaks. We have tools for doing this, but they are not documented. Prior to the PT2 release it would be good to document them. cc @voznesenskym

**Comment by voznesenskym** [comment]
Agreed. I have a draft of this, and of an internal developer facing doc I'm kinda slowly working on, usually while waiting for stuff to build. Let me prioritize this, I can share later today.

**Comment by voznesenskym** [comment]
Got distracted, I can get to it this week I think.
